src/services/routes.ts

```diff
@@ -0,0 +1,90 @@
+import { Scope } from '../models/connect';
+import Nylas from '../nylas';
+import AccessToken from '../models/access-token';
+import crypto from 'crypto';
+import { ExchangeCodeForTokenCallback } from '../config';
+
+export enum DefaultRoutes {
+  buildAuthUrl = '/generate-auth-url',
+  exchangeCodeForToken = '/exchange-mailbox-token',
+  webhooks = '/webhook',
```

Comment:
What do you think of adding the `/nylas` root route here to reduce the risk of route conflicts on a user's application?
```suggestion
  buildAuthUrl = '/nylas/generate-auth-url',
  exchangeCodeForToken = '/nylas/exchange-mailbox-token',
  webhooks = '/nylas/webhook',
```

Reply:
Implementation would be `app.post(DefaultRoutes.buildAuthUrl, async (req, res) => {` instead.

Unless we don't want to be that opinionated about this, but some pros to reserving a "root" level route.

Reply:
I think it's fine being that opinionated, because in that case they can just not use these values right?

Reply:
You guys think it is confusing to have `Routes` and `DefaultRoutes`? Maybe `DefaultPaths` or something instead?

Reply:
Sounds good:

- Defaulting to `/nylas` for paths as @krisgardiner suggested
- Allow overriding the paths in the ServerBinding config
- Renamed `DefaultRoutes` to `DefaultPaths` as @AaronDDM suggested